primes/zsh/compute.zsh

```diff
@@ -18,9 +18,12 @@ number=2
 echo begin
 while [[ $#primes -lt $PRIME_COUNT ]]; do
     if is_prime $number; then
-        echo $number
         primes+=($number)
     fi
     number=$((number + 1))
 done
+
+if [[ $BENCH_DEBUG == "true" ]]; then
+    printf '%s\n' "${primes[@]}"
```

Comment:
Why not use:

```zsh
printf "${primes[@]}\n"
```

Both versions print an extra newline when `PRIME_COUNT` is `0` but I guess we can assume it will be always at least `1` in tests.

Reply:
Fixed using the printing condition